assets/app/views/browse/deployment-config.html

```diff
@@ -5,7 +5,10 @@
     <div ng-if="!loaded">Loading...</div>
     <div class="tile" ng-if="deploymentConfig">
       <div> 
-        <h1>{{deploymentConfig.metadata.name}}</h1>
+        <h1>
+          {{deploymentConfig.metadata.name}}
+          <span ng-if="deploymentConfig.status.details.message" class="pficon pficon-warning-triangle-o" style="cursor: help;" data-toggle="popover" data-placement="right" data-trigger="hover" dynamic-content="{{deploymentConfig.status.details.message}}"></span>
```

Comment:
I asked, today it is always warnings

On Tue, Oct 20, 2015 at 4:36 PM, Sam Padgett <notifications@github.com>
wrote:

> In assets/app/views/browse/deployment-config.html
> <https://github.com/openshift/origin/pull/5251#discussion_r42550191>:
>
> > @@ -5,7 +5,10 @@
> >      <div ng-if="!loaded">Loading...</div>
> >      <div class="tile" ng-if="deploymentConfig">
> >        <div>
> > -        <h1>{{deploymentConfig.metadata.name}}</h1>
> > +        <h1>
> > +          {{deploymentConfig.metadata.name}}
> > +          <span ng-if="deploymentConfig.status.details.message" class="pficon pficon-warning-triangle-o" style="cursor: help;" data-toggle="popover" data-placement="right" data-trigger="hover" dynamic-content="{{deploymentConfig.status.details.message}}"></span>
>
> Is status.details.message always a warning?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5251/files#r42550191>.
>